titus-api/src/main/java/com/netflix/titus/api/FeatureActivationConfiguration.java

```diff
@@ -54,4 +54,10 @@
      */
     @DefaultValue("true")
     boolean isRelocationBinpackingEnabled();
+
+    /**
+     * Enable shared informer on Titus gateway
+     */
+    @DefaultValue("false")
+    boolean isKubeSharedInformerEnabled();
```

Comment:
I believe the shared informer is always enabled. Feature flag is for injecting container status. I think we should change the name of this method accordingly.